content/docs/1.6.0/references/settings.md

```diff
@@ -849,7 +849,7 @@ If it's hard to estimate the usage now, you can leave it with the default value,
 >  - Considering the possible number of new instance manager pods in a further system upgrade, this integer value ranges from 0 to 40.
 >  - One more set of instance manager pods may need to be deployed when the Longhorn system is upgraded. If current available CPUs of the nodes are not enough for the new instance manager pods, you need to detach the volumes using the oldest instance manager pods so that Longhorn can clean up the old pods automatically and release the CPU resources. And the new pods with the latest instance manager image will be launched then.
 >  - This global setting will be ignored for a node if the field "InstanceManagerCPURequest" on the node is set.
->  - After this setting is changed, all instance manager pods using this global setting on all the nodes will be automatically restarted. In other words, DO NOT CHANGE THIS SETTING WITH ATTACHED VOLUMES.
+>  - After this setting is changed, the v1 instance manager pod using this global setting will be automatically restarted without instances running on the v1 instance manager.
```

Comment:
I'll review this line after "without instances running on the v1 instance manager" is clarified.

Reply:
> "without instances running on the v1 instance manager"

Yes, like what you mentioned above.
The instance manager pod will be restarted only when no instances are running on the Instance Manager and the setting is not applied to the pod.

Reply:
```suggestion
>  - After the setting is changed, the V2 Instance Manager pods that use this setting are automatically restarted when no instances are running.
```

Reply:
Modified. Thanks.